packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx

```diff
@@ -76,3 +90,44 @@ export const getDocs: () => JSX.Element = () => {
     </>
   );
 };
+
+export const getFileSharingDocs: () => JSX.Element = () => {
+  return (
+    <>
+      <Title>File Sharing</Title>
+      <SingleLineBetaBanner />
+      <Description>Microsoft Teams enables file attachment sharing with the following permissions:</Description>
+      <Description>- Anyone</Description>
+      <Description>- Individuals within your organization</Description>
+      <Description>- Participants currently in this chat</Description>
+      <Description>- People who already have access</Description>
+      <Description>- Specific individuals you select</Description>
+      <Description>
+        Currently, Azure Communication Services only supports the "Anyone" and "People you choose" options. The “People
```

Comment:
Can we add a line something along the lines of `Microsoft Teams allows user to share file attachments as part of chat message, these files are stored in SharePoint or OneDrive that require authentication to access those files.`